### DISCUSSION TITLE
Move the time consuming script of ROS initialization as a seperate script and assign an icon on the desktop.

### DISCUSSION THREAD

**Discussion by wycc** [wycc/cgu-kubeflow#2] [General]
![image](https://github.com/wycc/cgu-kubeflow/assets/457473/2932e237-7be6-4db4-b9b2-6eca1c2acc32)

Make this part of script to a seperate script so that it won't block the start up script.

**Comment by wycc** [comment]
It will spend for two minutes in the first boot for now. If we do not assign enough CPU to it. It will take longer time.

**Comment by WinterBelieve** [comment]
I have written an independent script for ROS, named setup_catkin_ws.sh
